File: Angular Front End/src/app/book-table/comment.ts
export class Comment{
    bookId: string;
    name: string;
    review: string;
    rating: string;


  constructor(bookId: string, name: string, review: string, rating: string) {
    this.bookId = bookId;
    this.name = name;
    this.review = review
    this.rating = rating;
    
  }

 
}

File: Api/Book.Repository/Models/Comment.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Book.Repository.Models
{
    public class Comment
    {
        public int CommentId { get; set; }
        public string BookId { get; set; }

        public string Name { get; set; }

        public string Review { get; set; }

        public int Rating { get; set; }
    }
}
